Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -132,6 +132,20 @@ extension NSObject {
             traitSpecifiers.append(strings.tabTraitName)
         }
 
+        if accessibilityTraits.contains(.textEntry) {
+            if accessibilityTraits.contains(.scrollable) {
+                // This is being a UITextView/TextEditor
+                if let accessibilityValue = accessibilityValue, !accessibilityValue.isEmpty {
```

Comment:
Need to test this behavior, there was disagreement on previous pull request: https://github.com/cashapp/AccessibilitySnapshot/pull/58/files#r617273212